public/manifest.json

```diff
@@ -27,5 +27,7 @@
 
   "permissions": ["storage", "scripting", "activeTab"],
 
-  "host_permissions": ["https://api.codecov.io/*"]
+  "host_permissions": ["https://api.codecov.io/*"],
+
+  "optional_host_permissions": ["<all_urls>"]
```

Comment:
We need this permission to bypass cross origin restrictions for requests made from the extension to the codecov API